app/frontend/src/features/communities/discussion/DiscussionsListPage.tsx

```diff
@@ -78,10 +79,11 @@ export default function DiscussionsListPage({
         <CreateDiscussionForm
           communityId={community.communityId}
           onCancel={() => setIsCreatingNewPost(false)}
-          onPost={() => setIsCreatingNewPost(false)}
+          onPostSuccess={() => setIsCreatingNewPost(false)}
         />
       </Collapse>
       <div className={classes.discussionsContainer}>
+        {!isDiscussionsLoading && isDiscussionsFetching && <CircularProgress />}
```

Comment:
Can this be moved to next to the "New post" button? That way it won't cause content shifting.

We could also only display this if `isFetchedAfterMount` is true, then it will (I think?) only be displayed on invalidation and not clicking on the page. But maybe we don't want that behaviour anyway.

Reply:
Tried `isFetchedAfterMount` and didn't seem to be work, it stayed as true even when it's fetching after invalidation. Moved the loading state next to the "new post" button and yeah definitely less jumping around now